www/style.css

```diff
@@ -2357,8 +2322,6 @@ dl.mempool-histogram .bar:before {
 .sub-nav a{
   margin-right: 50px;
   color: #ffffff;
-  font-weight: bold;
-  line-height: 2.5;
 }
 
 .sub-nav .active {
```

Comment:
```suggestion
.sub-nav .active {
  color: #ffffff;
  position: relative;
}

.sub-nav .active::after {
  position: absolute;
  bottom: -10px;
  content: '';
  border-bottom: solid 2px #ffbb00;
  width: 100%;
  display: block;
}
```

Reply:
Thank you @Randy808 for pointing out that issue!